templates/plugin/salt_master.conf.erb

```diff
@@ -0,0 +1,58 @@
+# /etc/salt/master.d/foreman.config Example configuration
+#
+# This file summarizes configurations for the salt-master. Modify directories and
+# parameters to fit your setup. When you're done, remove the .example from the
+# filename so the salt-master will make use of it.
+# Have a look at the [Foreman Salt Plugin Documentation](https://theforeman.org/plugins/foreman_salt/) for detailed explanations.
+#
+# After editing this file, run the following command to active the changes:
+# $ systemctl restart salt-master
+
+
+##
+# Autosign
+autosign_grains_dir: <%= scope.lookupvar('::foreman_proxy::plugin::salt::autosign_grains_dir') %>
+autosign_file: <%= scope.lookupvar('::foreman_proxy::plugin::salt::autosign_file') %>
+# Uncomment the next line to make use of the autosign host name file (not recommended)
+# permissive_pki_access: True
+
+
+##
+# Node classifier
+master_tops:
+  ext_nodes: /usr/bin/foreman-node
+
+
+##
+# Pillar data access
+ext_pillar:
+  - puppet: /usr/bin/foreman-node
+
+
+##
+# Salt API access
+external_auth:
+  <%= scope.lookupvar('::foreman_proxy::plugin::salt::api_auth') %>:
+    <%= scope.lookupvar('::foreman_proxy::plugin::salt::api_username') %>:
+      - '@runner'
+
+rest_cherrypy:
+  port: 9191
+  ssl_key: <%= scope.lookupvar('::foreman_proxy::params::ssl_key') %>
+  ssl_crt: <%= scope.lookupvar('::foreman_proxy::params::ssl_cert') %>
```

Comment:
You should not use it from `foreman_proxy::params` since then you ignore user provided values.

Reply:
Is it better to use `@foreman_ssl_key` as I do now?